api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -184,11 +184,13 @@ type FelixConfigurationSpec struct {
 	// to Debug level logs.
 	LogDebugFilenameRegex string `json:"logDebugFilenameRegex,omitempty" validate:"omitempty,regexp"`
 
-	IPIPEnabled *bool `json:"ipipEnabled,omitempty" confignamev1:"IpInIpEnabled"`
+	// IPIPEnabled overrides whether Felix should configure an IPIP interface on the host. Deprecated as Felix now determines this based on the existing IP pools. [Default: nil (unset)]
```

Comment:
I'm not sure we can go as far as to say that these fields are deprecated on the v3 API.

The one thing we are confident about deprecating is their use directly within felix - felix should be considering the sum of these fields and the IP pools to determine encap state.

However, for an end-user, these fields are still something they can directly configure.

Reply:
Oh, I misunderstood, will fix. Thanks!